vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/PollOptionViewState.kt

```diff
@@ -51,4 +51,12 @@ sealed class PollOptionViewState(open val optionId: String,
                          val votePercentage: Double,
                          val isWinner: Boolean
     ) : PollOptionViewState(optionId, optionAnswer)
+
+    /**
+     * Represent a poll that is closed, votes will be hidden until the poll is ended.
```

Comment:
Maybe replace `closed` by `undisclosed` to avoid confusion between `close` and `ended`

Reply:
Right, done.